packages/web/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+    "extends": "../../tsconfig-base.json",
+    "include": ["src/**/*"],
+    "exclude": ["node_modules", "grpc/**/*"],
+    "rerefences": [
+        { "path": "../common"}
+    ],
+    "compilerOptions": {
+        "outDir": "./lib/src"
```

Would't it be nicer with simply

```suggestion
        "outDir": "./lib"
```